clang/include/clang/Basic/Attr.td

```diff
@@ -236,6 +236,10 @@ class DefaultIntArgument<string name, int default> : IntArgument<name, 1> {
   int Default = default;
 }
 
+class DefaultUnsignedArgument<string name, int default> : UnsignedArgument<name, 1> {
+  int Default = default;
```

Comment:
Why do you use `int` here if this is **unsigned** argument? I guess this should be `unsigned int`

Reply:
Maybe I mistake, but I think the TableGen has no unsigned type, but during processing Default static variable become unsigned anyway in C++ code.